addons/website_event/static/src/snippets/options.js

```diff
@@ -33,12 +34,16 @@ options.registry.WebsiteEvent = options.Class.extend({
     /**
      * @see this.selectClass for parameters
      */
-    displaySubmenu(previewMode, widgetValue, params) {
-        return this._rpc({
+    async displaySubmenu(previewMode, widgetValue, params) {
+        const rpcData = await this._rpc({
             model: 'event.event',
             method: 'toggle_website_menu',
             args: [[this.eventId], widgetValue],
         });
+        if (!rpcData) {
```

Comment:
I don't think "toggle_website_menu" returns anything, does it? Why this condition?